docs/sphinx/user_guide/feature/view.rst

```diff
@@ -82,6 +82,36 @@ accesses array entries with unit stride. The loop::
 
 access array entries with stride N :subscript:`n` * N :subscript:`(n-1)` * ... * N :subscript:`(j+1)`.
 
+MultiView
+^^^^^^^^^^^^^^^^
+
+A ``RAJA::MultiView`` object wraps an array-of-pointers,
+or a pointer-to-pointers, rather than a ``RAJA::View`` which wraps a single
+pointer or array. This allows a single ``RAJA::Layout`` to be applied to a
+large set of arrays, and encapsulates a set of arrays more easily.
+
+The instantiation of a MultiView works exactly like a standard View,
+except that it wraps an array-of-pointers::
+
+  int * myarr[2];
+  RAJA::MultiView< int, RAJA::Layout<2> > MView(myarr, N0, ... Nn);
+
+The MultiView can access individual arrays via the zero-th index::
+
+  MView( 0, x, y ); // accesses the (x,y)th element of the first array

```

Comment:
It would be helpful if the comment included the expected value of MView(0, x, y) for x=1 and y=4.  Or add a couple of extra concrete access examples here.

Reply:
Thanks, Steve! You're right, the example is a bit too lean. I've updated the documentation to address both of your comments.